data/json/monsters/nether.json

```diff
@@ -970,8 +970,8 @@
   {
     "id": "mon_shifting_mass",
     "type": "MONSTER",
-    "name": { "str": "shifting mass", "str_pl": "shifting masses" },
-    "description": "A small, shifting mass floating in midair.  It is not even real enough to have a stable shape.  You can see through it, not as if it were transparent, but as if it isn't quite fully there.",
+    "name": { "str": "shade" },
```

Comment:
I don't really understand the purpose of this change.

Reply:
Specifically the name, the description is good I just don't really understand the value in changing the name of a monster people already know.

Reply:
Tho why not add a new less abstract monster instead of demystifying this one

Reply:
purely personal preference; this is what I rolled with for a while and I liked the feeling of it. more specifically, the rewritten description says that it has a more established form than what the shifting mass has, so I gave it a different name since it's both no longer shifting and no longer a formless mass. I can reverse it either way, though, if it ends up causing contention

Reply:
I guess to reiterate I like your idea it just seems like it could be it's own seperate creature.

Reply:
The original version of this monster was also squidish.  

I changed it to be more abstract because I wanted Portal Storms when possible to emphasize that reality is breaking, these things are like glitches in reality more than mutant creatures.  

The current version is kinda dull though I agree and needs some polish.

Reply:
That's valid. Would you prefer I stick with the "glitch" version?